core/templates/services/utils.service.ts

```diff
@@ -33,11 +33,12 @@ export class UtilsService {
     return true;
   }
 
-  isString(input: any): boolean {
+  isString(input: Object): boolean {
```

Comment:
@jameesjohn `Object` is a generic interface that can be used for anything in ts. It would be wrong if we accessed some properties of this variable that are not present in the `Object` interface. For ex `length` but here we are just checking the type of the `input` variable, So, I think it's fine to declare it as` Object`.

Reply:
But yeah, it can be left as `any` as defining it as an Object may create a confusion.

Reply:
I think you can infer its type based on usage

Reply:
Yup, that is why I'm saying to change it's type to `Object`. Or we can keep it as `any` because this function is a generic function.

Reply:
I think what James is suggesting is that why can't we make it more specific than `Object`?

Reply:
Add a comment to explain.

Reply:
Ditto.

Reply:
Done